build/linux/Makefile

```diff
@@ -87,6 +87,12 @@ INSTALLER_TMPDIR := $(INTERMEDIATE_DIR)/installer_tmp
 
 # GO Source dir for custom fluent bit plugin
 GO_SOURCE_DIR := $(SOURCE_DIR)/plugins/go/src
+GO_SOURCE_INPUT_DIR := $(SOURCE_DIR)/plugins/go/input
+
+CMETRICS_VERSION = 0.6.0
```

Comment:
are you sure there is no way to remove this dependency? is this build or runtime dependency?

Reply:
there is no way to avoid this if we want to use calyptia/plugin for input